assemblies/features/base/src/main/filtered-resources/resources/bin/inc

```diff
@@ -44,16 +45,20 @@ detectOS() {
             ;;
         AIX*)
             aix=true
+            # For AIX, set an environment variable
+            export LDR_CNTRL=MAXDATA=0xB0000000@DSA
+            echo ${LDR_CNTRL}
             ;;
         OS400*)
             os400=true
             ;;
+        HP-UX*)
+            hpux=true
+            # For HP-UX, set an environment variable
+	    export PS_PREFIX="UNIX95= "
```

Comment:
Could you just fix the indentation with spaces and remove the leading tab please ?

Reply:
OK, done.